vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,10 +1,199 @@
+* Installation parameters considered by the operator all moved to the entando-operator-config ConfigMap.
+* Better support for cloud DBMS.
+* Updated libraries (Jetty, Keycloak, Spring) and the entire compatibility matrix.
+* Dropped support for suffix-mode installation. Only single-hostname is supported now.
+* Temporarily dropped support for cluster-level operator installation.
+* Removed EntandoClusterInfrastructure and EntandoCompositeApp CR.
+* k8s-service is now part of the base Entando infrastructure and therefore installed alongside the operator.
+* ComponentManager now communicates with k8s-service using service-to-service authentication, and strictly within the cluster network.
+* Entando Operator deployment is now simply called entando-operator.
+* k8s-service deployment dropped the application qualifier and is now simply called entando-k8s-service.
+* Fixed an intermittent issue with the operator that prevented installing new bundles until the operator was restarted.
+* Updated all the Entando base images for reliability and security.
+* Replaced the centos images to rocky images and rebased the derived images.
+* Updated the auto provisioned PostgreSQL image to version 14.
+* MySQL and PostgreSQL are now served by customized and updated entando-mysql and entando-postgres images.
+
+### Entando Bundler
 
-Entando implements the SQL versioning tool Liquibase to track, manage and deploy updates to databases. Refer to [Liquibase Migration](../reference/databases.md#liquibase-migration) to learn how the App Engine automates database schema management and modifications.
+* It’s now possible to extract protected assets.
+* Fixed several bugs in the extraction of pages and widgets.
+* Added support for categories.
+* Error handling has been improved.
+* It's now possible to generate the so-called system-level bundles (see ECR).
+* Extract/download optimized to only touch the relevant resources.
+* Libraries updated for reliability and security.
+
+### EOL Kubernetes Versions
+
+Entando7 drops official support for for OS3.11 and Kubernetes versions prior to 1.20.
+
+### Google Kubernetes Engine (GKE)
+
+* Postponed support to 7.0.1.
+
+### Azure Kubernetes Service (AKS)
+
+* Postponed support to 7.0.1.
+
+## Breaking Changes
+
+* The internal PostgreSQL version has been updated to version 14. Bundles that rely on the auto provisioning of PostgreSQL may (or may not) be affected.
+
+## Known Issues
+
+### Apple M1
+* Entando does not currently support ARM64 architecture and cannot be installed on newer Macs built with the Apple M1 processor.
+### ECG
+
+* The default "details" widget generated for an entity doesn't work out of the box.
+
+### ECR
+
+* An in-place upgrade from version 6.3.X to version 7 of the ECR is not possible, but a simple upgrade script will be provided. In-place upgrades from version 7 to future versions will be possible.
```

Comment:
@w-caffiero-entando or @firegloves is Liquibase implemented for the ECR/CM already? I don't really want to promise in-place upgrade after version 7.0.0 if it isn't implemented yet.

Reply:
@w-caffiero-entando confirmed, Liquibase is also in place for ECR/CM for 7.0.0

Reply:
@Lyd1aCla1r3 Let's try this "See the [Upgrade Steps](#upgrade-steps) below to upgrade a 6.3.2 ECR to 7.0.0. Liquibase support is now in place to enable automatic database upgrades going forward."